components/ArticleList.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react"; 
+import ArticleCard from "./ArticleCard"
+
+
+const ArticleList= () => {
+    const [isLoading, setIsLoading] = useState(true);
+    const [list , setList] = useState([]);
+
+
+
+    useEffect(() => {
+        console.log("useEffect triggered!")
+        fetch("https://nc-news-zcdp.onrender.com/api/articles")
+        .then((response) => response.json())
+        .then((body) => {
+            setList(body.articles);
+            setIsLoading(false);
+        });
+    }, []);
+
+if (isLoading) {
+    return <p> Loading... </p>
+}
+
+    return (
+        <section> 
+        <h2> NC Articles </h2>
+        <ul>
+            {list.map((article) => <ArticleCard key = {article.article_id}
+            title = {article.title}
+            topic = {article.topic}
+            author = {article.author}
+            published = {article.created_at}
```

Comment:
Optionally, you can format this date to be a little nicer by using a method such as [toUTCString()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toUTCString) 

(though you'd have to turn it into a new date object first, as that method is only available on dates!)